projects/kit/components/input-date-time/input-date-time.component.ts

```diff
@@ -314,10 +316,23 @@ export class TuiInputDateTimeComponent
         min: TuiDay,
         max: TuiDay,
         timeMode: TuiTimeMode,
+        dateFormat: TuiDateMode,
     ): TuiTextMaskOptions {
         return {
-            mask: [...TUI_DATE_MASK, ',', ' ', ...tuiCreateTimeMask(timeMode)],
-            pipe: tuiCreateAutoCorrectedDateTimePipe({value: day, min, max}, timeMode),
+            // TODO finish localization in {@link https://github.com/TinkoffCreditSystems/taiga-ui/issues/954 issue}
+            mask: [
```

Comment:
why can't we use `tuiCreateDateMask` here?

Reply:
sorry, i dont understand your question((

`tuiCreateDateMask` is already used here

Reply:
Oh, sorry, I fault (( I read wrong